integration-tests/src/main/java/com/datastax/oss/quarkus/dao/nameconverters/NameConverterEntity.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.quarkus.dao.nameconverters;
+
+import com.datastax.oss.driver.api.mapper.annotations.Entity;
+import com.datastax.oss.driver.api.mapper.annotations.NamingStrategy;
+import com.datastax.oss.driver.api.mapper.annotations.PartitionKey;
+
+@Entity
+@NamingStrategy(customConverterClass = TestNameConverter.class)
+public class NameConverterEntity {
```

Comment:
In the spirit of facilitating comprehension it might be better to rename this entity "ReflectionEntity" or something along those lines to clearly indicate what makes it unique.  Same argument applies for the entire chain of services and endpoints which ultimately lead to this class.

Reply:
It may be not a good solution, because in the future we may add more tests that validate other reflection calls - if we will rename it, in such a case there will be name clash. 
I think that the reader should start analyzing this code from:
```
@RegisterForReflection
public class TestNameConverter implements NameConverter 
```
and should notice the `@RegisterForReflection` that will give more insights about the purpose of this class.